roles/validate_http_store/tasks/main.yml

```diff
@@ -1,32 +1,40 @@
+---
+- name: Create a random name for the test file
+  ansible.builtin.set_fact:
+    vhs_test_file_name: "{{ lookup('password', '/dev/null length=12 chars=ascii_letters,digits') }}"
```

Comment:
I like the idea of using a random name, I believe you could keep it in the defaults and that way the var will not persist after the role.

Reply:
I tried that, but when it was defined as the default, it generated a new random string every time the variable was referenced.

Reply:
if it's ok to use only digits for the filename, there is an option to [generate idempotent random number](https://kottapar.medium.com/idempotent-random-port-in-ansible-82f536c88f74) with the seed.
```
# Generate an idempotent random port number for the dev_user in the range 1024–32767
dev_port: “{{ 32767 | random(1024, seed=dev_user) }}”
```